lib/RGBot.js

```diff
@@ -463,87 +935,130 @@ const RGBot = class {
     /**
      * Attempt to locate the nearest block of the given type within a specified range from the Bot.
      * @param blockType {string} The displayName or name of the block to find
-     * @param {object} options={} Optional parameters
+     * @param {object} [options={}] Optional parameters
      * @param {boolean} [options.partialMatch=false] Find blocks whose name / displayName contains blockType. (Ex. 'log' may find any of 'spruce_log', 'oak_log', etc.)
      * @param {boolean} [options.onlyFindTopBlocks=false] Will not return any blocks that are beneath another block
      * @param {number} [options.maxDistance=50] Find any Blocks matching the search criteria up to and including this distance from the Bot
      * @param {boolean} [options.skipClosest=false] Will attempt to locate the next-closest Block. This can be used to skip the closest Block when the Bot encounters an issue collecting it
      * @returns {Block | null}
+     *
+     * @deprecated Since 1.2.0, please use findBlocks(options, blockValueFunction, sortValueFunction)
```

Comment:
Once again, I don't think we need to deprecate this? I would prefer to use this more often I think.